### PR TITLE
Grammar time, again.

### PR DESCRIPTION
**Pull request by Belsima**
Grammar and makes ALL CAPS DESCRIPTION DEAR GOD WHY DO WE KEEP USING ALL CAPS use an appropriate comic sans instead. 

Just because ALL CAPS is frowned upon for players, so it should also be in code.

### CODE REVIEW

File: code/game/gamemodes/changeling/powers/shriek.dm [hunk 1]
```diff
@@ -20,19 +20,19 @@
 /mob/proc/changeling_resonant_shriek()
 	set category = "Changeling"
 	set name = "Resonant Shriek (20)"
-	set desc = "Emits a high-frequency sound that confuses and deafens humans, blows out nearby lights and overloads cyborg sensors."
+	set desc = "Emits a high-frequency sound that confuses and deafens organics, blows out nearby lights, and overloads synthetic's sensors."
```
Comment (Belsima) [inline code/game/gamemodes/changeling/powers/shriek.dm]: More than just humans.

File: code/game/gamemodes/changeling/powers/shriek.dm [hunk 2]
```diff
@@ -1,6 +1,6 @@
 /datum/power/changeling/resonant_shriek
 	name = "Resonant Shriek"
-	desc = "Our lungs and vocal chords shift, allowing us to briefly emit a noise that deafens and confuses the weak-minded biologicals and synthetics."
+	desc = "Our lungs and vocal cords shift, allowing us to briefly emit a noise that deafens and confuses the weak-minded."
```
Comment (Belsima) [inline code/game/gamemodes/changeling/powers/shriek.dm]: 'biologicals' refers to medications, and 'organics' would apply to the 'ling itself.

File: code/modules/power/gravitygenerator.dm [hunk 3]
```diff
@@ -94,7 +94,7 @@
 			else
 				dat += "<tt><font color=red>[A]</tt></font><br>"
 
-		dat += "<br><tt>Maintainence Functions:</tt><br>"
```
Comment (Belsima) [inline code/modules/power/gravitygenerator.dm]: maintaneince

File: code/game/gamemodes/changeling/powers/shriek.dm (hunk 1 above)
Comment (mistyLuminescence) [reply code/game/gamemodes/changeling/powers/shriek.dm]: synthetics'

File: code/modules/admin/NewBan.dm [hunk 4]
```diff
@@ -23,7 +23,7 @@ var/savefile/Banlist
 				.["desc"] = "\nReason: [Banlist["reason"]]\nExpires: [GetExp(Banlist["minutes"])]\nBy: [Banlist["bannedby"]][appeal]"
 		else
 			Banlist.cd	= "/base/[ckey][id]"
-			.["desc"]	= "\nReason: [Banlist["reason"]]\nExpires: <B>PERMENANT</B>\nBy: [Banlist["bannedby"]][appeal]"
+			.["desc"]	= "\nReason: [Banlist["reason"]]\nExpires: <B>PERMANANT</B>\nBy: [Banlist["bannedby"]][appeal]"
```
Comment (mistyLuminescence) [inline code/modules/admin/NewBan.dm]: perman**e**nt
you both got it wrong

File: code/modules/admin/NewBan.dm [hunk 5]
```diff
@@ -49,7 +49,7 @@ var/savefile/Banlist
 					else
 						.["desc"] = "\nReason: [Banlist["reason"]]\nExpires: [GetExp(Banlist["minutes"])]\nBy: [Banlist["bannedby"]][appeal]"
 				else
-					.["desc"] = "\nReason: [Banlist["reason"]]\nExpires: <B>PERMENANT</B>\nBy: [Banlist["bannedby"]][appeal]"
+					.["desc"] = "\nReason: [Banlist["reason"]]\nExpires: <B>PERMANANT</B>\nBy: [Banlist["bannedby"]][appeal]"
```
Comment (mistyLuminescence) [inline code/modules/admin/NewBan.dm]: permanent, E-A-E

File: code/modules/power/gravitygenerator.dm (hunk 3 above)
Comment (mistyLuminescence) [reply code/modules/power/gravitygenerator.dm]: that physically hurt to read

File: code/modules/admin/NewBan.dm (hunk 4 above)
Comment (Belsima) [reply code/modules/admin/NewBan.dm]: fuck

File: code/modules/admin/NewBan.dm (hunk 5 above)
Comment (Belsima) [reply code/modules/admin/NewBan.dm]: go away

File: code/modules/power/gravitygenerator.dm (hunk 3 above)
Comment (Belsima) [reply code/modules/power/gravitygenerator.dm]: mantainanince

File: code/game/gamemodes/changeling/powers/shriek.dm [hunk 6]
```diff
@@ -20,19 +20,19 @@
 /mob/proc/changeling_resonant_shriek()
 	set category = "Changeling"
 	set name = "Resonant Shriek (20)"
-	set desc = "Emits a high-frequency sound that confuses and deafens humans, blows out nearby lights and overloads cyborg sensors."
+	set desc = "Emits a high-frequency sound that confuses and deafens organics, blows out nearby lights, and overloads synthetics' sensors."
```
Comment (Belsima) [inline code/game/gamemodes/changeling/powers/shriek.dm]: skrell too!

File: code/modules/clothing/under/jobs/civilian.dm [hunk 7]
```diff
@@ -70,7 +70,7 @@
 
 /obj/item/clothing/under/rank/clown
 	name = "clown suit"
-	desc = "<i>'HONK!'</i>"
+	desc = "<i><font face='comic sans ms'>Honk!</i></font>"
```
Comment (Belsima) [inline code/modules/clothing/under/jobs/civilian.dm]: contentious tag

bikeshed tag

do not merge tag

requires staff input tag

**Pull request merged**